src/javascript/index.js

```diff
@@ -0,0 +1,65 @@
+const profileDetails = document.getElementById("aside");
+const header = document.getElementById("header");
+const footer = document.getElementById("footer");
+
+const username = "savannah-hayes";
+const USER_URL = `https://api.github.com/users/${username}`;
+const API_TOKEN = TOKEN || process.env.API_KEY
+
+const options = {
+  method: "GET",
+  headers: {
+    Authorization: `token ${API_TOKEN}`
+  }
+}
+
+const displayHtml = () => {
+  header.innerHTML = `
+    <div class="hamburger-wrapper">
+    <span class="hamburger"></span>
+    <span class="hamburger"></span>
+    <span class="hamburger"></span>
+    </div>
+    <h1>GitHub Tracker</h1>
+    <img src="./images/me.png" class="header__image" alt="image of savannah hayes">
+    `
+
+  footer.innerHTML = `
+    <p class="footer__text footer-flex"><img src="./images/github.png" class="footer__icon" alt="github icon">GitHub Tracker</p>
+    <ul class="footer__links footer-flex">
+    <li class="footer__link footer-flex">Terms</li>
+    <li class="footer__link footer-flex">Privacy</li>
+    <li class="footer__link footer-flex">Security</li>
+    <li class="footer__link footer-flex footer__link--hide">About</li>
+    </ul>
+  `
+}
+
+displayHtml()
+
+const displayProfileData = (profileData) => {
+  const { avatar_url, name, login, bio, followers, following, location } = profileData;
```

Comment:
Nice with destructuring!

Reply:
Oh I get it now! These are values from the data retrieved in the fetch, so you reused them to simply display in the html. This was so clever.